src/main/java/budgetbuddy/logic/commands/loancommands/LoanListCommand.java

```diff
@@ -17,23 +37,54 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all loans.\n"
             + "Parameters: "
-            + "[<person number ...>]"
-            + "[o|i] "
-            + String.format("[%s<d|a>]", PREFIX_SORT) + "\n"
+            + String.format("[%s|%s|", KEYWORD_LOAN_OUT, KEYWORD_LOAN_IN)
+            + String.format("%s|%s ...] ", KEYWORD_LOAN_UNPAID, KEYWORD_LOAN_PAID)
+            + "[" + PREFIX_PERSON + "PERSON ...] "
+            + "[" + PREFIX_AMOUNT + "AMOUNT ...] "
+            + "[" + PREFIX_DATE + "DATE ...] "
+            + "[" + PREFIX_DESCRIPTION + "DESCRIPTION ...] "
+            + String.format("[%s<%s|%s|%s>]\n", PREFIX_SORT, SORT_ARG_DATE, SORT_ARG_AMOUNT, SORT_ARG_PERSON)
             + "Example: "
-            + "loan list 1 2 o a";
+            + String.format("%s %s %s ", COMMAND_WORD, KEYWORD_LOAN_OUT, KEYWORD_LOAN_UNPAID)
+            + String.format("%sJohn %sMary %s%s ", PREFIX_PERSON, PREFIX_PERSON, PREFIX_DATE, DATE_EXAMPLE)
+            + String.format("%s%s", PREFIX_SORT, SORT_ARG_AMOUNT);
```

Comment:
Oh man, this looks horrible, but I'm not sure if we can improve this.

You know what we need? String interpolation. But no, we have to work in Java.

I should've done what I joked about in the past... write in Kotlin, then decompile the result and commit that.

Reply:
Rest assured that it was as horrible to type out as it is to gaze upon. :disappointed:

Reply:
Ah well it's fine. The only real issue here is whether the display will end up looking cluttered.

Reply:
The display in the GUI itself looks fine (it doesn't exceed the default width of the box). It just looks really gruesome in code.